client/Components/Map.js

```diff
@@ -10,17 +10,19 @@ const Wrapper = styled.div`
 
 `
 
-const Map = () => {
+const Map = ({locationData}) => {
+
+  const coordinates = [locationData.coordinates[1], locationData.coordinates[0]]
 
   useEffect(() => {
     mapboxgl.accessToken = 'pk.eyJ1IjoiY2hyaXN0dWxpbiIsImEiOiJja2pxM3l6bWkxZWxzMnFsZXB3ZTIzc3czIn0.7IMhPn6ShNGyxqpxSw-LCA';
```

Comment:
Move this to environment variable that is .gitignore d

Reply:
done